pkgs/tools/text/replace/makefile.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile	2021-11-29 16:32:12.292438858 +0800
++++ b/Makefile	2021-11-29 16:32:02.695427703 +0800
+@@ -19,7 +19,7 @@
+ ifeq ($(OS),Linux)
+ 
+ # Linux C compiler flags
+-CC = cc
++CC ?= cc
```

Comment:
This one did not require patching: https://github.com/NixOS/nixpkgs/pull/151295/commits/5f6a771d30bde115672b75552f0c2f0d1c80b0a5